plugins/git/plugin.vala

```diff
@@ -89,6 +90,29 @@ public class Marlin.Plugins.Git : Marlin.Plugins.Base {
             return;
         }
 
+        FileInfo info;
+        try {
+            info = directory.get_target_location ().query_filesystem_info (FileAttribute.FILESYSTEM_TYPE);
+        }
+        catch (GLib.Error error) {
+            if (!(error is IOError.CANCELLED)) {
+                warning ("GIT PLUGIN: Error querying %s filesystem info: %s", directory.uri, error.message);
+            }
+
+            info = null;
```

Comment:
Info will be set to NULL on error so we can return directly as we already know that we will return later on
```suggestion
            return;
```

Reply:
Yes, good point, you don't need the debug message in this case as you will get the warning message anyway.